src/Shell/Task/ExtractTask.php

```diff
@@ -122,7 +122,8 @@ protected function _getPaths()
             $response = $this->in($message, null, $defaultPath);
             if (strtoupper($response) === 'Q') {
                 $this->err('Extract Aborted');
-                return $this->_stop();
+                $this->_stop();
+                return;
```

Comment:
Based on static analysis it can't know what a method would return so it's understand :)